gef.py

```diff
@@ -2867,14 +2865,34 @@ def get_terminal_size():
     """Return the current terminal size."""
     if is_debug():
         return 600, 100
+    if platform.system() == "Windows":
+        from ctypes import windll, create_string_buffer
+        #stdin handle = -10
+        #stdout handle = -11
+        #stderr handle = -12
+        herr = windll.kernel32.GetStdHandle(-12)
```

Comment:
Please remove the comments as they don't provide any information about the code itself.
Something like 
```python
hStdErr = -12
herr = windll.kernel32.GetStdHandle(hStdErr)
```
is more explicit.